tests/modules/memberships/application/checkout-subscription-sync.test.ts

```
@@ -0,0 +1,428 @@
+import {
+	afterAll,
+	beforeAll,
+	beforeEach,
+	describe,
+	expect,
+	it,
+	vi,
+} from "vitest";
+import type { EmailGateway } from "@/modules/email";
```

Biome (2.4.4)

[error] This import is unused.

(lint/correctness/noUnusedImports)